AI/VCAI/VCAI.h

```diff
@@ -406,7 +402,6 @@ class VCAI : public CAdventureAI
 		h & visitableObjs;
 		h & alreadyVisited;
 		h & reservedObjs;
-		h & saving;

```

Comment:
do not forget to put version check/backward compatibility here, this should be done from very beginning

Reply:
As far as I see, old properties can be dropped with no harm. Either way, not sure how to handle it.

Reply:
Just realized I have no old savegame to test it, anyway.

Reply:
Nothing could be changed in serialization without harm.